service/common/build.gradle.kts

```diff
@@ -41,19 +44,21 @@ dependencies {
     exclude("com.sun.jersey", "jersey-core")
     exclude("com.sun.jersey", "jersey-server")
     exclude("com.sun.jersey", "jersey-servlet")
+    exclude("com.sun.jersey", "jersey-servlet")
+    exclude("io.dropwizard.metrics", "metrics-core")
   }
   implementation(libs.hadoop.hdfs.client)
 
-  compileOnly(libs.jakarta.annotation.api)
-  compileOnly(libs.jakarta.inject.api)
-  compileOnly(libs.jakarta.servlet.api)
-  compileOnly(libs.jakarta.validation.api)
-  compileOnly(libs.jakarta.ws.rs.api)
+  implementation(libs.jakarta.annotation.api)
+  implementation(libs.jakarta.enterprise.cdi.api)
+  implementation(libs.jakarta.inject.api)
+  implementation(libs.jakarta.servlet.api)
+  implementation(libs.jakarta.validation.api)
+  implementation(libs.jakarta.ws.rs.api)
```

Comment:
Considering this is a `service-common` module, do we need these libs in runtime classpaths?

Reply:
I will take a note to check if these would work with `compileOnly`.